lib/src/signed_video_h26x_sign.c

```diff
@@ -294,8 +294,6 @@ generate_sei_nalu(signed_video_t *self, uint8_t **payload, uint8_t **payload_sig
     uint8_t *payload_ptr = *payload;
 
     // Start writing bytes.
-    // Reset last_two_bytes before writing bytes
-    self->last_two_bytes = LAST_TWO_BYTES_INIT_VALUE;
```

Comment:
You still need to initialize this value. It is not taken from the buffer.
Either you initialize the temporary variable `self->last_two_bytes` like
```
self->last_two_bytes = LAST_TWO_BYTES_INIT_VALUE;
```
or you get it from the buffer (for which it is initialized already), like
```
self->last_two_bytes = self->last_two_bytes_buffer[self->payload_buffer_idx / 2];
```